### PR TITLE
Removing cosmos endpoint and fixing arm templates

### PR DESCRIPTION
**Pull request by tejaswip-msft**
# What
Fixing editableazuredeploy 
Removed managed identity resources from ARM template
Removed endpoint from docs and code

# Why

# How Tested
1. Tested by setting connection string and db name in env --> submitted results are reflected in db
2. Tested by setting connection string and db name in default config --> submitted results are reflected in db
3. Editable azure deploy flow 
4. Azure deploy flow

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by dailam-msft** [comment]
This change can be tested. Please add ss of successful deployment using this modified json in "edit template"

**Review by ralphgabrinao — DISMISSED** [review]

**Review by ralphgabrinao — DISMISSED** [review]

File: deploy/editableazuredeploy.json [hunk 1]
```diff
@@ -152,6 +149,9 @@
         "[resourceId('Microsoft.Web/serverfarms', variables('appServicePlanPortalName'))]",
         "[resourceId('Microsoft.Communication/communicationServices', parameters('communicationServiceName'))]"
       ],
+      "identity": {
```
Comment (ralphgabrinao) [inline deploy/editableazuredeploy.json:152]: No need for this now

File: server/src/utils/getCosmosDbConfig.ts [hunk 2]
```diff
@@ -1,30 +1,22 @@
...
-    return undefined;
-  }
 
-  if (cosmosDBConnectionString) {
-    cosmosDBConfig.connectionString = cosmosDBConnectionString;
-  }
+  const cosmosDBConfig: CosmosDBConfig = {
+    dbName: process.env[VV_COSMOS_DB_NAME] ?? (defaultConfig.cosmosDb?.dbName as string),
+    connectionString:
+      process.env[VV_COSMOS_DB_CONNECTION_STRING] ?? (defaultConfig.cosmosDb?.connectionString as string)
```
Comment (ralphgabrinao) [inline server/src/utils/getCosmosDbConfig.ts:14]: Can we just do this once? This part is repeated in line 9 process.env[VV_COSMOS_DB_CONNECTION_STRING] ?? (defaultConfig.cosmosDb?.connectionString

**Comment by dailam-msft** [comment]
Let's rename the PR as the objective has changed

File: client/src/components/postcall/PostCallOneQuestionPoll.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ export const PostCallOneQuestionPoll: React.FunctionComponent<PostCallOneQuestio
 
   const submitSurveyResponse = async (): Promise<void> => {
     await submitSurveyResponseUtil(props.acsUserId, pollResponse, props.callId);
-    window.location.replace('/book');
+    // window.location.replace('/book');
```
Comment (dailam-msft) [inline client/src/components/postcall/PostCallOneQuestionPoll.tsx:25]: Why are we commenting this?

Comment (tejaswip-msft) [reply client/src/components/postcall/PostCallOneQuestionPoll.tsx:25]: Sorry that was by mistake.  will revert back

File: server/src/databaseHandlers/surveyDBHandler.ts [hunk 4]
```diff
@@ -1,31 +1,20 @@
...
-  return new CosmosClient(config.cosmosDb.connectionString);
-};
-
 export const createSurveyDBHandler = (config: ServerConfigModel): SurveyDBHandler | undefined => {
+  console.log('in createSurveyDBHandler');
   if (config.postCall?.survey.type && config.postCall.survey.type === 'onequestionpoll') {
+    console.log('post call is one question poll');
     if (config.cosmosDb) {
-      const cosmosClient = createCosmosClient(config);
+      console.log('cosmosDb is defined');
```
Comment (ralphgabrinao) [inline server/src/databaseHandlers/surveyDBHandler.ts:16]: Can remove the console logs here

File: server/src/databaseHandlers/surveyDBHandler.ts [hunk 5]
```diff
@@ -1,31 +1,18 @@
...
-    };
-
-    return new CosmosClient(cosmosClientOptions);
-  }
-
-  return new CosmosClient(config.cosmosDb.connectionString);
-};
-
 export const createSurveyDBHandler = (config: ServerConfigModel): SurveyDBHandler | undefined => {
+  console.log('in createSurveyDBHandler');
```
Comment (ralphgabrinao) [inline server/src/databaseHandlers/surveyDBHandler.ts:12]: One more console log to remove here :D

File: deploy/azuredeploy.json [hunk 6]
```diff
@@ -63,10 +63,7 @@
     "postCallSurveyOneQuestionPollAnswerPlaceholder": "",
     "postCallSurveyOneQuestionPollSaveButtonText": "Continue",
     "cosmosDbAccountName": "[concat('db-account-', toLower(parameters('appServiceName')))]",
-    "roleDefinitionName": "[concat('sql-role-definition-', toLower(parameters('appServiceName')))]",
-    "cosmosDbName": "[concat('cosmos-db-', toLower(parameters('appServiceName')))]",
-    "roleDefinitionId": "[guid('sql-role-definition-', resourceId('Microsoft.DocumentDB/databaseAccounts', variables('cosmosDbAccountName')))]",
-    "roleAssignmentId": "[guid(variables('roleDefinitionId'), resourceId('Microsoft.DocumentDB/databaseAccounts', variables('cosmosDbAccountName')))]"
+    "cosmosDbName": "[concat('cosmos-db-', toLower(parameters('appServiceName')))]"
```
Comment (rvenugopal-msft) [inline deploy/azuredeploy.json:66]: Where is the conn string being fetched and set in the arm template?

**Review by ralphgabrinao — DISMISSED** [review]

File: server/src/defaultConfig.json [hunk 7]
```diff
@@ -20,8 +20,7 @@
     }
   },
   "cosmosDb": {
-    "endpoint": "Set the endpoint to authenticate using Managed Identity method",
-    "connectionString": "Or, set the connection string. If both values are defined, endpoint takes precedence",
+    "connectionString": "Connection string of your cosmos DB account",
```
Comment (ralphgabrinao) [inline server/src/defaultConfig.json:23]: Super tiny nitpick: Change to `Cosmos` for consistency

File: docs/post-call-survey.md [hunk 8]
```diff
@@ -42,8 +42,7 @@ VV_COSMOS_DB_NAME = "Name of the Cosmos Database"
         }
     },
     "cosmosDB": {
-        "endpoint": "Set the endpoint to authenticate using Managed Identity method",
-        "connectionString": "Or, set the connection string. If both values are defined, endpoint takes precedence",
+        "connectionString": "Connection string of your cosmos DB account",
```
Comment (ralphgabrinao) [inline docs/post-call-survey.md:45]: Same here `Cosmos`

File: docs/post-call-survey.md [hunk 9]
```diff
@@ -42,8 +42,7 @@ VV_COSMOS_DB_NAME = "Name of the Cosmos Database"
         }
     },
     "cosmosDB": {
```
Comment (ralphgabrinao) [inline docs/post-call-survey.md:44]: cosmosDb

**Review by ralphgabrinao — DISMISSED** [review]

**Review by hansung-msft — DISMISSED** [review]

**Review by hansung-msft — APPROVED** [review]

**Review by ralphgabrinao — APPROVED** [review]

**Pull request merged**